Resources/config/user_context.xml

```diff
@@ -39,5 +40,9 @@
             <argument />
             <argument />
         </service>
+
+        <service id="fos_http_cache.user_context.anonymous_request_matcher" class="FOS\HttpCacheBundle\UserContext\AnonymousRequestMatcher">
+            <argument />
```

Comment:
it should be ``<argument type="collection" />`` to pass an empty array rather than an empty string